jwcrypto/jwk.py

```diff
@@ -549,12 +550,84 @@ def import_from_pyca(self, key):
         else:
             raise InvalidJWKValue('Unknown key object %r' % key)
 
+    def import_from_pem(self, data, password=None):
+        """Imports a key from data loaded from a PEM file.
```

Comment:
I don't like the fact that the method attempts to load public and private keys. As an application developer I'd like to have more control over the key type and have dedicated methods to load either a private or a public key.